include/RAJA/exec-agency/raja_agency.hxx

```diff
@@ -93,35 +96,62 @@ using agency_sequential_exec = agency_base<
   decltype(agency::seq)
 >;
 
-template <typename AGENT, typename WORKER>
-struct agency_taskgraph_base { };
+#if defined(RAJA_ENABLE_OPENMP)
+using agency_omp_parallel_exec = agency_base<
+  agency::parallel_agent, 
+  decltype(agency::omp::par)
+>;
+#endif
 
-using agency_taskgraph_sequential_segit = agency_taskgraph_base<
-  agency::sequenced_agent,
-  decltype(agency::seq)
+#if defined(RAJA_ENABLE_CUDA)
+// Wrapper functor because we can't template on the overloaded function grid
+struct AgencyCudaGrid {
```

Comment:
We can't template on agency::cuda::grid because it is templated itself; this seemed like the simplest solution.